uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/DocumentTest.php

```diff
@@ -50,18 +50,6 @@ function testCreateDocumentWithNoSource() {
   /**
    * @expectedException CRM_Exception
    */
-  function testCreateDocumentWithNoAssignee() {
```

Comment:
I reverted the deletion and remove the `expectException` but I don't have testing set up locally so I can't be sure if these tests are passing now.